packages/product-react/src/Tabs/Tabs.scss

```diff
@@ -0,0 +1,66 @@
+$easing: cubic-bezier(0.694, 0.0482, 0.335, 1);
+$ray-color-black: black;
+$ray-color-blue-50: blue;
```

Comment:
Are we not able to use `Ray-core` variables here instead 😛?

Reply:
yes pls!